app/Actions/Fortify/CreateNewUser.php

```diff
@@ -75,8 +75,8 @@ public function create(array $input): User
             ]
         )->validate();
 
-        Cookie::queue('theme', 'light');
         Cookie::queue('locale', $input['locale']);
+        Cookie::queue('theme', Cookie::get('theme', 'system'));
```

Comment:
Does this actually need to be queued. Shouldn't it already be set if you are getting it?

Reply:
I see, you're queueing it in case there was no cookie found and it used the fallback value.